<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Virtual pet</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>470</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="feed_Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="walk_Button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="play_Button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="doctor_Button">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Feed</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Walk</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Doctor</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>971</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>501</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>540</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>610</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>480</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hunger</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>550</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mood</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>620</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Health</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>470</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>480</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Energy</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>974</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSleep">
   <property name="icon">
    <iconset>
     <normaloff>V-Pet Game/Sourses/sleep.png</normaloff>V-Pet Game/Sourses/sleep.png</iconset>
   </property>
   <property name="text">
    <string>Sleep</string>
   </property>
  </action>
  <action name="actionplay">
   <property name="icon">
    <iconset>
     <normaloff>V-Pet Game/Sourses/play.png</normaloff>V-Pet Game/Sourses/play.png</iconset>
   </property>
   <property name="text">
    <string>play</string>
   </property>
  </action>
  <action name="actionWalk">
   <property name="icon">
    <iconset>
     <normaloff>V-Pet Game/Sourses/walk.png</normaloff>V-Pet Game/Sourses/walk.png</iconset>
   </property>
   <property name="text">
    <string>Walk</string>
   </property>
  </action>
  <action name="actionFood">
   <property name="icon">
    <iconset>
     <normaloff>V-Pet Game/Sourses/food.png</normaloff>V-Pet Game/Sourses/food.png</iconset>
   </property>
   <property name="text">
    <string>Food</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset>
     <normaloff>V-Pet Game/Sourses/help.png</normaloff>V-Pet Game/Sourses/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
